src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -31,14 +32,15 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(Authentication auth, @RequestParam Long movieSessionId) {
```

Comment:
@RequestParam should go before Authentication
```suggestion
    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
```